presto-docs/src/main/sphinx/include/PrestoThriftService.thrift

```diff
@@ -0,0 +1,305 @@
+namespace java.swift com.facebook.presto.connector.thrift.api
```

Comment:
Can we remove this? We don't need to dictate this as it's a concern for code generators.